### DISCUSSION TITLE
ECU zip - latest database ?

### DISCUSSION THREAD

**Discussion by AndyIreland** [cedricp/ddt4all#902] [Q&A]
I have 6356 ECU's loaded in database - I wondered if there is a latest bigger ECU Database file out there?

![image](https://github.com/cedricp/ddt4all/assets/49172217/e50b2a46-9b55-43db-b55a-57a021b0ed73)

**Comment by KarelSvo** [comment]
It's not the size that matters, but the quality. I only packed the ECUs relevant to my vehicles into ecu.zip.
And previously i repaired the files with errors. The errors are marked red in the ddt2000 Builder. I also removed dangerous  commands that can damage an ecu. The last known update is from 2022-10-25T18:27:52 and contains 3086 ecu files.

**Comment by radjool** [comment]
I need dacia duster 2023 ecu.zip file but couldn't find it. could you share  your database with me please

**Comment by aliuzunoglu** [comment]
Can someone please add this ecu.zip file to the source code or somewhere that can be found easily?

**Comment by Furtif** [comment]
https://github.com/cedricp/ddt4all/discussions/845#discussion-5264316
